docs/tutorials/images.rst

```diff
@@ -100,87 +102,84 @@ Armed with this information, it's possible to create a new image like this::
 
     primes = [2, 3, 5, 7, 11, 13, 17, 19]
 
-
 .. note::
 
-    Numbers don't need to be quoted since they represent a value (rather than a
-    string of characters). It's the difference between ``2`` (the numeric value
-    2) and ``"2"`` (the character/digit representing the number 2). Don't worry
-    if this doesn't make sense right now. You'll soon get used to it.
+    Liczby nie potrzebują być w cudzysłowie dopóki reprezentują wartość (w przeciwieństwie
+    do ciągów znaków). Jest różnica pomiędzy ``2`` (numeryczna wartość 2) i ``"2"``
+    (znak/cyfra reprezentująca liczbę 2). Nie martw się jeżeli nie widzisz w
+    tym sensu teraz. Z czasem będzie to dla Ciebie oczywiste.
 
-It's even possible to store different sorts of things in the same list::
+Jest nawet możliwe przechowywanie różnych rodzajów rzeczy w tej samej liście::
 
     mixed_up_list = ["hello!", 1.234, Image.HAPPY]
 
-Notice that last item? It was an image!
+Zwróć uwagę na ostatni element? To był obrazek!
 
-We can tell MicroPython to animate a list of images. Luckily we have a
-couple of lists of images already built in. They're called ``Image.ALL_CLOCKS``
-and ``Image.ALL_ARROWS``::
+Możemy powiedzieć MicroPythonowi aby animował listę obrazków. Szczęśliwie mamy
+już kilka wbudowanych list obrazków. Są to ``Image.ALL_CLOCKS`` i
+``Image.ALL_ARROWS``::
 
     from microbit import *
 
     display.show(Image.ALL_CLOCKS, loop=True, delay=100)
 
-As with a single image, we use ``display.show`` to show it on the
-device's display. However, we tell MicroPython to use ``Image.ALL_CLOCKS`` and
-it understands that it needs to show each image in the list, one after the
-other. We also tell MicroPython to keep looping over the list of images (so
-the animation lasts forever) by saying ``loop=True``. Furthermore, we tell it
-that we want the delay between each image to be only 100 milliseconds (a tenth
-of a second) with the argument ``delay=100``.
+Używamy ``display.show`` do pokazania listy obrazków na ekranie urządzenia
+tak, jak w przypadku pojedynczego obrazka. Jednak mówimy MicroPythonowi użyj
+``Image.ALL_CLOCKS`` i on rozumie, że potrzebuje pokazać każdy obrazek z listy
+jeden po drugim. Mówimy też MicroPythonowi aby użył pętli dla listy obrazków
+(tak więc animacja trwa nieskończenie) przez ``loop=True``. Ponadto mówimy
+jemu, że chcemy aby opóźnienia pomiędzy każdym obrazkiem były tylko 100
+milisekund (0,1 sekundy) w argumencie ``delay=100``.
 
-Can you work out how to animate over the ``Image.ALL_ARROWS`` list? How do you
-avoid looping forever (hint: the opposite of ``True`` is ``False`` although
-the default value for ``loop`` is ``False``)? Can you change the speed of the
-animation?
+Czy możesz domyślić się jak animować obrazki z listy ``Image.ALL_ARROWS``?
+Jak możesz uniknąć nieskończonego powtarzania (podpowiedź: przeciwieństwem do
+``True`` (ang. prawdziwy) jest ``False`` (ang. fałszywy) chociaż domyślna
```

Comment:
prawdziwy → prawda
fałszywy → fałsz

Reply:
W buttons.rst True tłumaczone jest: (ang. prawdziwe)

Reply:
Masz rację, późno było. Prawa to oczywiście Truth i tak dalej.